libraries/botbuilder-dialogs-adaptive/src/actions/continueConversationLater.ts

```diff
@@ -0,0 +1,122 @@
+/**
+ * @module botbuilder-dialogs-adaptive
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    BoolExpression,
+    BoolExpressionConverter,
+    Expression,
+    StringExpression,
+    StringExpressionConverter,
+    ValueExpression,
+    ValueExpressionConverter,
+} from 'adaptive-expressions';
+import { Activity, ActivityEventNames, ActivityTypes, QueueStorage, TurnContext } from 'botbuilder-core';
+import {
+    Converter,
+    ConverterFactory,
+    Dialog,
+    DialogConfiguration,
+    DialogContext,
+    DialogTurnResult,
+    DialogTurnStateConstants,
+} from 'botbuilder-dialogs';
+
+export interface ContinueConversationLaterConfiguration extends DialogConfiguration {
+    disabled?: boolean | string | Expression | BoolExpression;
+    date?: string | Expression | StringExpression;
+    value?: unknown | ValueExpression;
+}
+
+/**
+ * Action which schedules a conversation to be continued later by writing an EventActivity(Name=ContinueConversation) to a queue.
+ */
+export class ContinueConversationLater<O extends object = {}>
+    extends Dialog<O>
+    implements ContinueConversationLaterConfiguration {
+    public static $kind = 'Microsoft.ContinueConversationLater';
+
+    /**
+     * Gets or sets an optional expression which if is true will disable this action.
+     */
+    public disabled: BoolExpression;
```

Comment:
Do these values all need defaults?

Reply:
The default value of a `BoolExpression` is false, so we don't have to set default value for the `disabled` property. I should mark it as optional. For other properties, I didn't see any default values since `date` is a required property and `value` is undefined by default.